### PR TITLE
feat: support multiple modes of export 

### PR DESCRIPTION
**Pull request by rlespinasse**
Add possibility to also export only changed files or since a specific git reference instead of the current full export

BREAKING CHANGE: by default, only the changed drawio files will  be exported

### CODE REVIEW

**Comment by rlespinasse** [comment]
Replacing #2

File: .github/workflows/drawio-export-action.yml [hunk 1]
```diff
@@ -31,12 +41,16 @@ jobs:
...
+          action-mode: all
 
   drawio-export-action-release:
     runs-on: ubuntu-latest
     needs:
       - drawio-export-action-os-testing
+    concurrency:
+      group: drawio-export-action-release-${{ github.ref }}
     steps:
+      - uses: technote-space/auto-cancel-redundant-workflow@v1
```
Comment (Ameausoone) [inline .github/workflows/drawio-export-action.yml]:
```suggestion
```
so this one is useless now, no ?

**Review by Ameausoone — APPROVED** [review]
I validate, but I don't understand so much that I'm not sure it's relevant :)

File: .github/workflows/drawio-export-action.yml (hunk 1 above)
Comment (rlespinasse) [reply .github/workflows/drawio-export-action.yml]: Good catch

**Pull request merged**